hilti/runtime/include/types/stream.h

```diff
@@ -1389,7 +1389,14 @@ class View {
 };
 
 inline std::ostream& operator<<(std::ostream& out, const View& x) {
-    out << escapeBytes(x.data().str());
+    Bytes data;
+    try {
+        data = x.data();
+    } catch ( hilti::rt::MissingData& ) {
+        data.append("<missing data>");
+    }
+
+    out << escapeBytes(data.str());
```

Comment:
As discussed, we should change this to insert `<gap>` (or so) into the rendered data, instead of fully replacing it.